examples/src/main/java/io/dapr/examples/invoke/grpc/HelloWorldClient.java

```diff
@@ -70,10 +55,6 @@ public static void main(String[] args) throws Exception {
       } catch (StatusRuntimeException e) {
         logger.log(Level.WARNING, "RPC failed: {0}", e.getStatus());
       }
-    } finally {
```

Comment:
Do we still want this?

Reply:
No, because the channel is now encapsulated in the DaprClient, which is closed automatically by the try() statement in the end. The DaprClient itself will then close all downstream resources that need to be closed, including the gRPC channel.

Another example of why using the DaprClient is also useful for those that want to make gRPC proxy invocations in Java.